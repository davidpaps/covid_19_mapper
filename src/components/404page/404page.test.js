import React from "react";
import Page from "./404page";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("404 Page", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<Page />)));

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(1);
  });

  it("should render a <img />", () => {
    expect(wrapper.find("img").length).toEqual(1);
  });

  it("renders correctly", () => {
    const component = renderer.create(<Page />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
