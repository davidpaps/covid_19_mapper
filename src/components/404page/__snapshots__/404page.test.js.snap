// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Page should render correctly 1`] = `
<div
  id="error"
>
  <h1>
    Page Not Found Page
  </h1>
  <br />
  <img
    alt=""
    src="https://cdn3.iconfinder.com/data/icons/science-116/64/virus-lab-scientist-biology-cell-medical-512.png?v=2"
  />
</div>
`;
